src/containers/CreateAccount/CreateAccount.js

```diff
@@ -50,16 +50,24 @@ export class CreateAccount extends Component {
             )
         } else {
             return(
-                <form onSubmit={this.validateInput}>
-                    <label htmlFor='name'>Name</label>
-                    <input name='name' value={this.state.name} id='name' onChange={this.handleChange}/>
-                    <label htmlFor='email'>E-mail</label>
-                    <input name='email' value={this.state.email} id='email' onChange={this.handleChange}/>
-                    <label htmlFor='password'>Password</label>
-                    <input name='password' value={this.state.password} id='password' onChange={this.handleChange}/>
-                    { (errorStatus !== '' && errorStatus !== 'Input not successful') && <p>Email has already been used</p>}
-                    { (errorStatus === 'Input not successful') && <p>All fields must be filled in to create an account</p>}
-                    <button>Create Account</button>
+                <form onSubmit={this.validateInput} className='form-container'>
+                    <p className='main-text'>SIGN UP FOR FREE</p>
+                    <p className='sub-text'>Start tracking your favorite movies today!</p>
+                    <div className='name-container'>
+                        <label htmlFor='name'><i class="fas fa-user"></i></label>
+                        <input name='name' value={this.state.name} id='name' onChange={this.handleChange} placeholder='Full name'/>
+                    </div>    
+                    <div className='email-container'>
+                        <label htmlFor='email'><i class="far fa-envelope"></i></label>
+                        <input name='email' value={this.state.email} id='email' onChange={this.handleChange} placeholder='E-mail address'/>
+                    </div>
+                    <div className='password-container'>   
+                        <label htmlFor='password'><i class="fas fa-lock"></i></label>
+                        <input name='password' value={this.state.password} id='password' onChange={this.handleChange} placeholder='Password (between 5-10 characters)'/>
+                        { (errorStatus !== '' && errorStatus !== 'Input not successful') && <p>Email has already been used</p>}
```

Comment:
The form looks really good. The icons next to the input fields are a great addition. I did see that when an error message pops up on the CreateAccount form, it displays next to the password. What if we move the two error messages on line 67 and 68 outside of the div 'password-container'? That way they'll show below the password input.
<img width="1285" alt="screen shot 2019-02-05 at 8 16 07 am" src="https://user-images.githubusercontent.com/40586291/52284783-53056480-2922-11e9-8c55-99ec410fd779.png">